assets/js/partners/contact.js

```diff
@@ -0,0 +1,129 @@
+window.clearCaptchaError = () => {
+  const error = document.getElementById('captcha-error-message');
+  if (error.textContent) {
+    error.textContent = '';
+    error.classList.add('display-none');
+  }
+};
+
+function fetchTextInput(id) {
+  return document.querySelector(`#contact-us-form #${id}`).value;
+}
+
+function fetchSelectInput(name) {
+  return document.querySelector(`#contact-us-form select[name="${name}"]`).value;
+}
+
+async function submitPartnerContact(event) {
+  event.preventDefault();
+
+  const debug = Array.prototype.slice.apply(document.getElementsByName('debug'))[0];
```

Comment:
Do we want to prevent a real submission if `debug` is enabled? It looks to me that the submission would always go through.

Reply:
Nah I think it's fine, it's been helpful to test in the preview site